includes/Vendor/SetupWizard.php

```diff
@@ -147,9 +147,10 @@ public function setup_wizard_footer() {
      */
     public function dokan_setup_introduction() {
         $dashboard_url = dokan_get_navigation_url();
+        $setup_wizard_message = dokan_get_option( 'setup_wizard_message', 'dokan_general', 'default' );
```

Comment:
third parameter of dokan_get_option() should contain the actual message, we need to include this because if admin didn't click the save button vendor will see this message. Rewrite this code as below
```
$default_message = wp_kses( __( 'Thank you for choosing The Marketplace to power your online store! This quick setup wizard will help you configure the basic settings. <strong>It’s completely optional and shouldn’t take longer than two minutes.</strong>', 'dokan-lite' ), [ 'strong' => [] ] );

$setup_wizard_message = dokan_get_option( 'setup_wizard_message', 'dokan_general', $default_message );
```